pkg/registry/seedmanagement/managedseed/strategy.go

```diff
@@ -165,5 +165,12 @@ func ShootNameTriggerFunc(obj runtime.Object) string {
 		return ""
 	}
 
+	return GetShootName(managedSeed)
+}
+
+func GetShootName(managedSeed *seedmanagement.ManagedSeed) string {
```

Comment:
```suggestion
func getShootName(managedSeed *seedmanagement.ManagedSeed) string {
```

Reply:
It's used outside of this package, see https://github.com/stoyanr/gardener/blob/34bef2b51fa1c8083aa01ab22227353ada296259/pkg/registry/seedmanagement/managedseed/storage/tableconvertor.go#L87

Reply:
OK, missed that. Then I would only suggest to add a doc string as it's exported. Rest lgtm.

Reply:
Thanks, done.